src/pretix/plugins/stripe/payment.py

```diff
@@ -1938,3 +1947,25 @@ def _payment_intent_kwargs(self, request, payment):
                 },
             }
         }
+
+
+class StripeTwint(StripeRedirectMethod):
+    identifier = 'stripe_twint'
+    verbose_name = _('TWINT via Stripe')
+    public_name = 'TWINT'
+    method = 'twint'
+    confirmation_method = 'automatic'
+    explanation = _(
+        'This payment method is available to users of the Swiss app TWINT. Please have your app '
+        'ready.'
+    )
+
+    def is_allowed(self, request: HttpRequest, total: Decimal=None) -> bool:
+        return super().is_allowed(request, total) and request.event.currency == "CHF" and total < Decimal("5000.00")
```

Comment:
According to stripe docs maximum amount is 5.000 – so I read it as included.
```suggestion
        return super().is_allowed(request, total) and request.event.currency == "CHF" and total <= Decimal("5000.00")
```